src/Store/context.js

```diff
@@ -0,0 +1,22 @@
+import React, { createContext, useContext, useReducer } from 'react'
```

Comment:
this file can be renamed as storeContext.js as other file is /Product/productReducer.js. Consistency with naming convention